src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -50,6 +50,7 @@ public sealed class VsSolutionRestoreService : IVsSolutionRestoreService, IVsSol
         private const string RestoreSources = nameof(RestoreSources);
         private const string RestoreFallbackFolders = nameof(RestoreFallbackFolders);
         private const string AssetTargetFallback = nameof(AssetTargetFallback);
+        private const string RestoreAdditionalProjectFallbackFoldersExcludes = nameof(RestoreAdditionalProjectFallbackFoldersExcludes);
         private const string RestoreAdditionalProjectFallbackFolders = nameof(RestoreAdditionalProjectFallbackFolders);
         private const string RestoreAdditionalProjectSources = nameof(RestoreAdditionalProjectSources);
```

Comment:
I would say the improvement there would be more about being consistent among CLI/VS rather than perf improvement. 
But I had looked into it too, it'd need to end up in something like Common.